docs/p12-limitation.rst

```diff
@@ -0,0 +1,272 @@
+.. _p12-limitation:
+
+Configure terminated HTTPS load balancer with PKCS12 certificate bundle
+=======================================================================
+F5 Neutron Lbaas Dashboard support SSL offloading with PEM certificate. When use F5 Neutron Lbaas Dashboard to create or update a certificate for listener, user need to input ``server certificate``, ``intermediate certificate``, ``private key`` and ``passphrase`` into F5 Neutron Lbaas Dashboard.
+
+F5 Neutron Lbaas Dashboard does not support load balancer SSL offloading function with PKCS 12 format certificates.
+User need to manually extract PKCS 12 bundle before using F5 Neutron Lbaas Dashboard.
+
+1. To extract a PKCS 12 bundle via openssl tool.
+
+.. code-block:: bash
+
+  # extract certificate without encrypt private key
+  openssl pkcs12 -in certificate.pfx -out certificate.txt -nodes
+
+  # extract certificate with encrypt private key
+  openssl pkcs12 -in certificate.pfx -out certificate.txt
+
+2. Make sure the extracted certificates are PEM format by verifying each certificate.
+
+.. code-block:: bash
+
+  # verify the certificate PEM format
+  openssl x509 -in name.pem -noout -text
+
+3. Copy corresponding certificates and private key from extracted file to F5 Neutron Lbaas Dashboard.
+4. For the encrypted private key extracted from PKCS 12 file, user also need to input the passphrase of private key into F5 Neutron Lbaas Dashboard.
+
+Example: Create A PKCS 12 Certificate
```

Comment:
There's no need to tell the user how to create a PKCS cert bundle. If they didn't already know how to do this, they wouldn't be reading this document. Take out this entire section.

Reply:
agree